src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -431,7 +431,12 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
                         }
                         else
                         {
-                            WamAdapters.ThrowExceptionFromWamError(result, _logger);
+                            _logger?.WarningPii(
+                            $"[WamBroker] Could not sign out user {account.Username} - error: {result.Error}",
+                            $"[WamBroker] Could not sign out user, error: {result.Error}");
+
+                            string errorMessage = $"{result.Error} (error code : {result.Error.ErrorCode})";
+                            throw new MsalServiceException("wam_sign_out_failed", errorMessage);
                         }
```

Comment:
Also at line 420, I don't think that exception is useful. If an account is not found, let's just move on.